app/src/main/res/layout/item_chat.xml

```diff
@@ -24,25 +23,25 @@
         android:layout_marginRight="26dp"
         android:background="@drawable/chat_message_background"
         android:orientation="vertical"
-        android:paddingTop="4dp"
         android:paddingBottom="4dp"
         android:paddingLeft="8dp"
-        android:paddingRight="8dp">
+        android:paddingRight="8dp"
+        android:paddingTop="4dp">
```

Comment:
Dude, that auto-formatting code again. Is it an automatic thing or are you using ctrl + alt + L?